recipes/cudatoolkit/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+cuda_compiler_version:           # [linux64]
+  - 11.0                         # [linux64]
+docker_image:                    # [linux64]
+  - condaforge/linux-anvil-cuda  # [linux64]
```

Comment:
I don't think we need this image if we're maintaining the same approach of downloading and extracting the binary package